src/widget/wspinnybase.cpp

```diff
@@ -106,6 +107,10 @@ WSpinnyBase::~WSpinnyBase() {
 #endif
 }
 
+bool WSpinnyBase::shouldDrawVinylQuality() const {
+    return m_bVinylActive && m_bSignalActive && m_bDrawVinylSignalQuality;
+}
```

Comment:
```suggestion
bool WSpinnyBase::shouldDrawVinylQuality() const {
#if defined __VINYLCONTROL__
    return m_bVinylActive && m_bSignalActive && m_bDrawVinylSignalQuality;
#else
    return false;
#endif
}
```
This could simplify the code which calls `shouldDrawVinylQuality()`

Reply:
Are you suggesting to removed the ifdefs there? Makes sense. Actually, this will never become true if __VINYLCONTROL__ is not defined, but I guess it's more clear like this.